components/style/themes/default.less

```diff
@@ -461,3 +461,13 @@
 @slider-dot-border-color-active:      tint(@primary-color, 50%);
 @slider-disabled-color:               @disabled-color;
 @slider-disabled-background-color:    @component-background;
+
+// Collapse
+// ---
+@collapse-header-padding:             12px 0 12px 40px;
+@collapse-header-bg:                  @background-color-light;
+@collapse-content-padding-top:        @padding-md;
+@collapse-content-padding-right:      @padding-md;
+@collapse-content-padding-bottom:     @padding-md;
+@collapse-content-padding-left:       @padding-md;
```

Comment:
Just @collapse-padding, I don't think we need to customize padding for each different direction.

Reply:
I would like that too, but see above comment.

If I don't put top/bottom padding on the inner `box` element inside `component`, the animation gets very chunky and gross.

Reply:
ah, but I didn't try putting all of the padding on the `box` instead of on the `component` until just now.

putting all the padding on the `box` works fine. 

Making this change now.